dream_layer_backend/create_sample_grid.py

```
@@ -0,0 +1,170 @@
+        ("sample_003.png", (512, 512), (100, 100, 255), "Blue Abstract"),   # Blue
+        ("sample_004.png", (512, 512), (255, 255, 100), "Yellow Still Life"), # Yellow
+    ]
+    
+    for filename, size, color, title in sample_images:
+        img = Image.new("RGB", size, color)
+        draw = ImageDraw.Draw(img)
+        
+        # Add some artistic elements
+        try:
+            font = ImageFont.truetype("arial.ttf", 32)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)